lightning/src/ln/channelmonitor.rs

```diff
@@ -775,11 +761,6 @@ pub struct ChannelMonitor<ChanSigner: ChannelKeys> {
 	pending_htlcs_updated: Vec<HTLCUpdate>,
 	pending_events: Vec<events::Event>,
 
-	// Thanks to data loss protection, we may be able to claim our non-htlc funds
-	// back, this is the script we have to spend from but we need to
-	// scan every commitment transaction for that
-	to_remote_rescue: Option<(Script, SecretKey)>,
```

Comment:
Note: I'm already cutting out a lot of code based on `to_remote_output` after the #562 patchset, so rebasing this on top should even simplify change further.